lib/addons/p5.dom.js

```diff
@@ -1965,15 +1965,20 @@
   /**
    *
    * Sets the width and height of the element. AUTO can be used to
-   * only adjust one dimension. If no arguments given returns the width and height
-   * of the element in an object.
+   * only adjust one dimension at a time. If no arguments are given, it
+   * returns the width and height of the element in an object. In case of
+   * elements which need to be loaded, such as images, it is recommended
+   * to call the function after the element has finished loading.
    *
    * @method size
    * @return {Object} the width and height of the element in an object
    * @example
    * <div class='norender'><code>
-   * var div = createDiv('this is a div');
+   * const div = createDiv('this is a div');
    * div.size(100, 100);
+   * const img = createImg('assets/laDefense.jpg', () => {
```

Comment:
I would suggest making both of these `let` instead of `const`. even though they are not modified again in this example, it is not uncommon that users might do so.